crucible-syntax/src/Lang/Crucible/Syntax/Concrete.hs

```diff
@@ -160,6 +161,26 @@ instance Semigroup (ExprErr s) where
 instance Monoid (ExprErr s) where
   mempty = TrivialErr (OtherPos "mempty")
 
+instance PP.Pretty (ExprErr s) where
+  pretty =
+    \case
+      TrivialErr p ->
+        "Trivial error at" PP.<+> PP.viaShow p
+      Errs e1 e2 ->
+        PP.vcat ["Multiple errors:" , PP.pretty e1 , PP.pretty e2]
+      DuplicateAtom p a ->
+        "Duplicate atom `" PP.<> PP.pretty a PP.<> "` at" PP.<+> PP.viaShow p
+      DuplicateLabel p l ->
+        "Duplicate label `" PP.<> PP.pretty l PP.<> "` at" PP.<+> PP.viaShow p
```

Comment:
Similarly, use `enclose` here. (We might consider defining `` backticks = enclose "`" "`" `` for symmetry with the existing `parens` and `brackets` combinators in `prettyprinter`.)